smack-extensions/src/main/java/org/jivesoftware/smackx/geoloc/GeoLocationManager.java

```diff
@@ -83,4 +88,53 @@ public static boolean isGeoLocationMessage(Message message) {
         return GeoLocation.from(message) != null;
     }
 
+    /**
+     * Returns the geolocation node in PubSub.
+     * 
+     * @return the geolocation PubSub node
+     * @throws InterruptedException
+     * @throws NotConnectedException
+     * @throws XMPPErrorException
+     * @throws NoResponseException
+     */
+    public LeafNode getNode()
+            throws NoResponseException, XMPPErrorException, NotConnectedException, InterruptedException {
+        PubSubManager pubSubManager = PubSubManager.getInstance(connection());
+
+        try {
+            pubSubManager.createNode(GeoLocation.NAMESPACE);
+        } catch (Exception e) {
```

Comment:
@Flowdalic because if the node already exists, the createNode() throws an exception but the processing should continue to get it